docs/manual/config/index.rst

```diff
@@ -127,6 +127,9 @@ configuration variables that control specific aspects of Qtile's behavior:
         - smart: automatically focus if the window is in the current group
 
         - never: never automatically focus any window that requests it
+    * - ``window_activation_rules``
+      - ``[]``
+      - A list of Match objects of windows that will always be focused when the _NET_ACTIVE_WINDOW message is sent by applications.
```

Comment:
I wonder: can we make `focus_on_window_activation` take a function, so that people can define this stuff on their own?

Reply:
@tych0 Should `focus_on_window_activation` only take a function or either a function or a string with the old behaviors? (so old configs don't break)

Reply:
Yeah, we'll want old configs to stay working, so I think a string or a function should be allowable.